style.css

```diff
@@ -0,0 +1,41 @@
+.calculator{
+    width: 25%;
+    height: 40%;
+    margin: 0 auto;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-auto-rows: minmax(30px, 80px);
+    grid-gap: 5px;
+    font-size: 1.5em;
+    
```

Comment:
Let's clean this file up by removing the extra blank lines inside of the curly braces for all rules.